Documentation/INSTALLATION.md

```diff
@@ -2,7 +2,7 @@
 
 - Have a Fastly API Key in-hand with edit privileges
 - Have a service behind Fastly
-- Have WAF enabled for your account
+- Have [Fastly WAF (Legacy)](https://docs.fastly.com/en/guides/web-application-firewall-legacy) enabled for your account. (WAF-NG users should use the [API](https://developer.fastly.com/reference/api/waf/) / [Terraform provider](https://registry.terraform.io/providers/fastly/fastly/latest/docs/resources/service_waf_configuration) instead)
```

Comment:
i believe this is just talking about the feature flag. You can use the waflyctl to provision a WAF for any particular service. 

Feature flags would be: `WAF` and `WAF NG`

Reply:
I believe the internal version has some WAF-NF functionality, not the public release. The API / Terraform is the intended way to control WAF-NG from what i've discussed internally. Happy to be wrong here, but if not I want the repo to be clear about it :)

Reply:
Would it be worth linking to the docs that review how to setup the NG WAF instead of just the API docs? 
https://docs.fastly.com/en/guides/web-application-firewall#enabling-the-fastly-waf

Reply:
Thanks for the comment Squee. Ask discussed via Slack. We'll keep the `/en/guides/web-application-firewall-legacy` URL as it has the note on legacy vs NG on that page. 

Merging.